docs/_links.md

```diff
@@ -18,4 +17,38 @@ This endpoint checks to see if a link object exists for a given code.
 ### Query Parameters
 
 Parameter | Default | Description
---------- | ------- | ----------- 
+--------- | ------- | -----------
+
+## Post a Link
+
+```shell
+curl -XPOST "https://api.dc01.gamelockerapp.com/shards/na/link/{player_id}" \
+  -H "Authorization: Bearer <api-key>" \
+  -H "Accept: application/vnd.api+json"
+```
+
+This endpoint creates a PlayerLink object if the verification code matches the one provided by the game.
+
+### HTTP Request
+
+`POST https://api.dc01.gamelockerapp.com/link/{player_id}`
+
+### Query Parameters
+
+Parameter | Default | Description
+--------- | ------- | -----------
+code      |         | The verification code
+
+## PlayerLink
+
+```json
+{
+  "attributes": {
+      "playerId": "fb374a7b-78be-4fcc-83ed-6a532a8a6f55",
+      "shardId": "na",
+      "titleId": "semc-vainglory"
```

Comment:
Let's mention here what `titleId` is. It's obvious to those of us who're already working with the gamelocker API, but it may be less obvious as to why this is here to new developers, especially since we're removing `X-TITLE-ID`.